src/content/docs/errors-inbox/errors-inbox.mdx

```diff
@@ -49,7 +51,33 @@ Ready to get started? Make sure you have a [New Relic account](https://newrelic.
 
 Access to log management capabilities is limited by user type. See [user type capabilities](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type#user-type-capabilities).
 
-## Set up errors inbox [#get-started]
+## Use errors inbox directly from scoped views [#scoped]
```

Comment:
A couple things:

1.   I have noticed that capitalization is inconsistent across this entire page.   It seems to be either `errors inbox` or `Errors inbox`.  I think it should be the latter or both words should be capitalized. 
1.   I would say "... from entity scoped..."